.github/workflows/main.yml

```
@@ -131,3 +131,33 @@ jobs:
         shell: msys2 {0}
         run: |
           python .ci/ci-check.py
+
+  package-grokker:
+    runs-on: ubuntu-latest
+    needs: build
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: jeremyd2019/package-grokker
+      - uses: actions/download-artifact@v3
+        id: artifacts
+        continue-on-error: true
+        with:
+          name: MINGW64-packages
+          path: artifacts
+      - name: Install stuff
+        if: steps.artifacts.outcome == 'success'
+        run: |
+          python -m venv --system-site-packages venv
+          . venv/bin/activate
+          pip install -r requirements.txt
+          deactivate
+      - name: Grok packages
+        if: steps.artifacts.outcome == 'success'
+        run: |
+          . venv/bin/activate
+          for pkg in artifacts/*.pkg.tar.*; do
+            pkgname=$(basename "$pkg" | rev | cut -d- -f4- | rev)
+            python -u package-update-check.py -e MINGW64 -p "${pkgname}" @PKG@ "file://$(pwd)/${pkg}"
+          done
```

I think I've got the package-grokker code in a better state for this now.  Invocation changed because now it can process a directory of artifacts rather than needing to be called in a loop.

```suggestion
          python -u package-update-check.py -v -e mingw64 artifact_dir ./artifacts
```